addons/addon-base-raas/packages/base-raas-services/lib/aws-accounts/aws-cfn-service.js

```diff
@@ -305,6 +350,76 @@ class AwsCfnService extends Service {
     }
   }
 
+  async checkPendingAccounts(requestContext) {
+    const awsAccountsService = await this.service('awsAccountsService');
+    const accounts = await awsAccountsService.list();
+    const pendingAccountIds = _.map(
+      _.filter(accounts, acct => acct.permissionStatus === 'PENDING'),
+      'id',
+    );
+
+    const auditLog = {};
+    const newStatus = {};
+
+    const processor = async awsAccountId => {
```

Comment:
`processor` -> `onboardAccount`

Let's give it a more descriptive name than `processor`

Reply:
👍🏻